### ISSUE TITLE
error after 1 epoch

### ISSUE THREAD

**Issue by 853498747** [AntiAegis/Human-Segmentation-PyTorch#3]
when i run train.py unet_config
after trained 1 epoch,error shows:

Traceback
train.py line 101
       trainer.train()
base_trainer.py line95
       result = self.train_epoch(epoch)
trainer/trainer.py line98
      self.writer_train.add_image('train/output',make_grid(.....))
torchvision/utils.py ,line66, in make_grid
    norm_range(tensor,range)
torchvision/utils.py ,line60, in norm_range
   norm_ip(t,t.min,max=max)
torchvision/utils.py ,line53, in norm_ip
  img.clamp_(argument 'min' must be Number ,not tensor)
 
can you help me ,my own dataset input is rgb ,mask is 1 channel (0,1)

**Comment by AntiAegis** [comment]
It seems you customized the trainer.py.
Could you paste the trainer.py file here so that I can see what should be fixed?

**Comment by 853498747** [comment]
I have not change anything in trainer.py. I just change the picture path   config_UNet.json.
My label(mask) picture is one channel (0,1),shoud i make some changes?

**Comment by AntiAegis** [comment]
Hello @853498747, did you install the same version of PyTorch?
Which is your version of PyTorch?
